src/aesthetics.jl

```diff
@@ -169,7 +171,7 @@ function assert_aesthetics_undefined(who::AbstractString, aes::Aesthetics, vars:
     defined_vars = intersect(Set(vars), defined_aesthetics(aes))
     if !isempty(defined_vars)
         error(@sprintf("The following aesthetics are defined but incompatible with %s: %s\n",
-                       who, join(undefined_vars, ", ")))
+                       who, join(defined_vars, ", ")))
```

Comment:
Ah. Nice catch!

I looked into the `precompile.jl` code and I was convinced that it's a relic of pre-0.4 julia. I ripped it all out and as far as I can tell it doesn't introduce any regressions and introduces a dramatic speed in precompilation. I have a PR here: #958